packages/component/src/hooks/internal/useRegisterFocusTranscript.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react';
```

Comment:
For all new hooks, could you add a short description at the top for what it's doing? sometimes the names are not enough to determine.

Reply:
Added comments to several `useRegister*`. Later, I should write a meta-hook that refactor them together.

```js
// This hook is for registering and unregister a callback, used by <BasicTranscript>.
// When called, the callback should scroll the transcript in a relative motion, for example, page up and down.
```